FAQ.md

```diff
@@ -88,3 +88,11 @@ See for example [this StackExchange question](https://unix.stackexchange.com/a/1
 
 See also https://github.com/coreos/ignition/issues/585
 
+## Q: What happens when I use `rpm-ostree override replace` to replace an RPM?
+
+When a package is replaced in this fashion, it will remain in place through any subsequent upgrades.
+
+While this can be helpful for short-term fixes, it is important to remember that the package replacement
```

Comment:
Can hyperlink to https://github.com/openshift/machine-config-operator/issues/945

Reply:
Thanks, done!